ssh/proxy.go

```diff
@@ -491,6 +491,35 @@ func (c *connection) sendAuthReq() error {
 	if err != nil {
 		return err
 	}
+
+	// The server may choose to send a SSH_MSG_EXT_INFO at this point (if we
+	// advertised willingness to receive one, which we always do) or not. See
+	// RFC 8308, Section 2.4.
+	extensions := make(map[string][]byte)
+	if len(packet) > 0 && packet[0] == msgExtInfo {
+		var extInfo extInfoMsg
+		if err := Unmarshal(packet, &extInfo); err != nil {
```

Comment:
append handle the first read packet because the first message type is 7(extinfo) now. not 6.